src/libraries/Microsoft.PowerFx.LanguageServerProtocol/Protocol/ScalarRange.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+namespace Microsoft.PowerFx.LanguageServerProtocol.Protocol
+{
+    public class ScalarRange
+    {
+        public ScalarRange(int start, int end)
+        {

```

Comment:
nit: you don't need the constructor. Serializer would be creating this object during deserialization and you won't need to construct this manually. remove the constructor

Reply:
Sure, if needed for testing, we can add later.